dynamiqs/result.py

```diff
@@ -144,11 +144,12 @@ def _str_parts(self) -> dict[str, str | None]:
 
 class PropagatorResult(Result):
     @property
-    def propagators(self) -> QArray:
+    def propagators(self) -> Array:
```

Comment:
Can we convert the object back to a qarray in the propagator solver? (and remove the changes in this file)